packages/strapi-plugin-upload/admin/src/components/NavLink/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * HeaderNavLink
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import getTrad from '../../utils/getTrad';
+// import pluginId from '../../pluginId';
+import Wrapper from './Wrapper';
+
+/* istanbul ignore next */
+function HeaderNavLink({ isDisabled, to, isActive, onClick }) {
```

Comment:
`isDisabled` and `isActive`. It is a bit confusing for me.

Reply:
Also, you can change this function component to arrow function component.

Reply:
Well the `isActive` prop is also used in the content type builder so I kept the naming.